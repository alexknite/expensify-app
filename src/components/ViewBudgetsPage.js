import React from 'react';
import BudgetsSummary from './BudgetsSummary';
import BudgetList from './BudgetList';
import BudgetListFilters from './BudgetListFilters';

const ViewBudgetsPage = () => (
  <div>
    <BudgetsSummary />
    <BudgetListFilters />
    <BudgetList />
  </div>
);

export default ViewBudgetsPage;
