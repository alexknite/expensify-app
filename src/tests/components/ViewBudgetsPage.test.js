import React from 'react';
import { shallow } from 'enzyme';
import ViewBudgetsPage from '../../components/ViewBudgetsPage';

test('should render ViewBudgetsPage correctly', () => {
  const wrapper = shallow(<ViewBudgetsPage />);
  expect(wrapper).toMatchSnapshot();
});
