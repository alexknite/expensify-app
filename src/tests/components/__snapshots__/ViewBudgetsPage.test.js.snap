// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ViewBudgetsPage correctly 1`] = `
<div>
  <Connect(BudgetsSummary) />
  <Connect(BudgetListFilters) />
  <Connect(BudgetList) />
</div>
`;
